src/models/editor.interface.ts

```diff
@@ -1,6 +1,7 @@
 import type { EditorArguments } from './editorArguments.interface';
 import type { EditorValidationResult } from './editorValidationResult.interface';
-
+import type { GridOption } from '../models/gridOption.interface';
+import type { Column } from '../models/column.interface';
```

Comment:
you are already inside the `/models` folder, there's no need to get out and get back in on your import path. 

Also now that you have a few imports, you can probably use the barrel index import instead, I typically do that when having a few

```suggestion
import type { Column, EditorArguments, EditorValidationResult, GridOption } from './index';
```